diesel_codegen/src/database_url.rs

```diff
@@ -29,8 +29,8 @@ fn load_dotenv_file() -> Result<(), String> {
 
 #[cfg(not(feature = "dotenv"))]
 fn load_dotenv_file() -> Result<(), String> {
-    Err(String::from("The dotenv feature is required to use strings starting \
-        with `dotenv:`"))
+    Err(String::from("diesel_codegen must be compiled with the dotenv feature \
+        to use strings starting with `dotenv:`"))
```

Comment:
using "substitute" seems to me to be exposing the internals of how the library works a little too much, although I admittedly haven't looked at much of the code and don't know what the standard is. How about "to correctly interpret strings starting with" or "to allow passing strings starting with"? I'm also happy to go with your suggestion if you think it's best.